presto-docs/src/main/sphinx/functions/map.rst

```diff
@@ -12,6 +12,12 @@ The ``[]`` operator is used to retrieve the value corresponding to a given key f
 Map Functions
 -------------
 
+.. function:: all_key_match(x(K,V), function(K, boolean)) -> boolean
```

Comment:
how do we handle all_key_match(map(), x -> false) here?

Reply:
It will return true. All these edge cases will follow the same pattern as standard ALL_MATCH function for array

Reply:
the function is called `all_keys_match` (keys is plural). can you fix the documentation?

Reply:
@rschlussel nice catch. Fixing it.